src/json-patch-duplex.js

```diff
@@ -70,6 +70,31 @@ var jsonpatch;
                 return false;
         }
     }
+    function _deepClone(value) {
+        // from here https://jsperf.com/deep-copy-vs-json-stringify-json-parse/25 (recursiveDeepCopy)
+        var clone;
+        var i;
+        if (typeof value !== 'object') {
+            return value;
+        }
+        if (!value) {
+            return value;
+        }
```

Comment:
these two ifs can be combined to `if (typeof value !== 'object' || value === null)`